broker/api/v10.py

```diff
@@ -78,3 +78,80 @@ def submission_status(submission_id):
 @rest.get('/submissions/<submission_id>/log')
 def submission_log(submission_id):
     return u.render(api.submission_log(submission_id))
+
+""" Return the visualizer URL of a specific submission.
+
+    Normal response codes: 200
+    Error response codes: 400
+"""
+
+@rest.get('/submissions/<submission_id>/visualizer')
+def submission_visualizer(submission_id):
+    return u.render(api.submission_visualizer(submission_id))
+
+""" Add a new cluster reference in the Asperathos section.
+
+    Normal response codes: 202
+    Error response codes: 400, 401
+"""
+@rest.post('/submissions/cluster')
+def add_cluster(data):
+    return u.render(api.add_cluster(data)) 
+
+""" Add a certificate to a cluster reference in the Asperathos section.
+
+    Normal response codes: 202
+    Error response codes: 400, 401
+"""
+@rest.post('/submissions/cluster/<cluster_name>/certificate')
+def add_certificate(cluster_name, data):
+    return u.render(api.add_certificate(cluster_name, data)) 
+
+""" Delete a certificate to a cluster reference in the Asperathos section.
+
+    Normal response codes: 202
+    Error response codes: 400, 401
+"""
+@rest.post('/submissions/cluster/<cluster_name>/certificate/<certificate_name>/delete')
```

Comment:
any particular reason for POSTing to a `/delete` endpoint instead of using HTTP DELETE method on the regular resource endpoint? same for other delete endpoints in this PR.

Reply:
Modification made in commit 540b521953feea4e73758f6d8173a1c58e9e65be